internal/api/rbac/policy_rules.go

```diff
@@ -49,27 +52,25 @@ func BuildNormalizedPolicyRulesMap(
 ) (map[string]rbacv1.PolicyRule, error) {
 	rulesMap := make(map[string]rbacv1.PolicyRule)
 	for _, rule := range rules {
-		for _, group := range rule.APIGroups {
-			group = strings.TrimSpace(group)
-			if group == "*" {
-				return nil, fmt.Errorf("wildcard APIGroup is not allowed")
```

Comment:
Now we don't even _look_ at the group since we auto-detect the correct one or fail of we don't recognize the resource type.